### PR TITLE
add missing prezi changelog entries

### PR DESCRIPTION
**Pull request by azaroth42**
Only two missing entries that I could find from RC3.

Assigning to @zimeon for his double-check.

### CODE REVIEW

**Comment by zimeon** [comment]
https://preview.iiif.io/api/1831_prezi_changelog/api/presentation/3.0/change-log/

File: source/api/presentation/3.0/change-log.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The Presentation API classes in version 2.1.1 had both `members` to allow for mi
...
 
-The `license` property was renamed to the more general `rights` to accommodate both rights statements and usage licenses. The value was further constrained to allow only Creative Commons URIs, RightsStatements.org URIs, and URIs registered as extensions. This additional constraint is to allow clients to treat the property as an enumeration rather than free text, and implement URI specific behavior. See issues [#644](https://github.com/IIIF/api/issues/644) and [#1479](https://github.com/IIIF/api/issues/1479)
+The `license` property was renamed to the more general `rights` to accommodate both rights statements and usage licenses. The value was further constrained to allow only http Creative Commons URIs, https RightsStatements.org URIs, and any URI registered as an extension. This additional constraint is to allow clients to treat the property as an enumeration rather than free text, and implement URI specific behavior. See issues [#644](https://github.com/IIIF/api/issues/644), [#1479](https://github.com/IIIF/api/issues/1479), and [#1874](https://github.com/IIIF/api/issues/1874) which was approved by [trc#32](https://github.com/IIIF/trc/issues/32).
```
Comment (mikeapp) [inline source/api/presentation/3.0/change-log.md]: http/https are formatted `http` and `https` in the spec

Comment (zimeon) [reply source/api/presentation/3.0/change-log.md]: Fixed with ce6ed91 -- thanks

**Comment by azaroth42** [comment]
:+1: to simeon's changes

**Pull request merged**